task/thermal/src/control.rs

```diff
@@ -357,15 +360,21 @@ impl Default for OneSidedPidState {
 }
 
 /// This corresponds to states shown in RFD 276
+///
+/// All of our temperature arrays contain, in order
+/// - I2C temperature inputs (read by this task)
+/// - Dynamic temperature inputs (read by another task and passed in)
 enum ThermalControlState {
     /// Wait for each sensor to report in at least once
```

Comment:
Is this comment still accurate? I.e., do we wait for all the dynamic inputs to report at least once?

Reply:
Every `dynamic_input` that's not `None` has to report in at least once; if they are `None`, then they are skipped during `Boot` and become `TemperatureReading::Inactive` during `Running`.

Double-checking this logic would be appreciated!

Reply:
Yep, I read over that later and forgot to come back to this comment. It looks good to me.